### PR TITLE
website: add sponsor section on homepage

### PR DESCRIPTION
**Pull request by adrien-f**
## Changes

* website: add sponsor section

image

### CODE REVIEW

**Comment by bwplotka** [comment]
Awesome! How we should proceed here? Ask our users to provide logos to this PR? @adrien-f @domgreen

**Comment by domgreen** [comment]
I would personally merge this PR ... then ask people to open their own PRs against master. Means we dont end up stacking up too many and delaying this one

**Comment by domgreen** [comment]
Maybe add a small documentation saying `How to add my logo as a Thanos user?` Then out line in a number of bullets how to do it

**Comment by adrien-f** [comment]
Can you give me an Improbable-Eng logo to get started ?

**Comment by bwplotka** [comment]
@adrien-f sending you on DM

**Comment by jakekeeys** [comment]
can we host them inside the repository?

File: website/layouts/index.html [hunk 1]
```diff
@@ -52,7 +52,33 @@ <h4 class="font-weight-bold">Downsampling & Compaction</h4>
...
+            <div class="img-sponsor">
+                <a href="https://improbable.io" rel="nofollow" target="_blank"><img src="{{ "/logos/improbable.png" | relURL }}" class="img-fluid" alt="Improbable"></a>
+            </div>
+        </div>
+    </div>
+    {{ if $.Site.Data.sponsors.sponsors }}
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Used by</h3></div>
+    </div>
+    <div class="row">
```
Comment (bwplotka) [inline website/layouts/index.html]: Wow you automated this <3 

Can we add link to `how-to-contribute-to-docs.md#Logos` with message something like "Want to be added to list? Follow this"

File: website/layouts/index.html [hunk 2]
```diff
@@ -52,7 +52,33 @@ <h4 class="font-weight-bold">Downsampling & Compaction</h4>
         </div>
     </div>
 </div>
-<div class="container-fluid">
+<div class="container py-5">
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Created by</h3></div>
```
Comment (bwplotka) [inline website/layouts/index.html]:
```suggestion
        <div class="col-12"><h3 class="text-center">Founded by</h3></div>
```

File: website/hugo.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ relativeURL: true
 contentDir: "docs-pre-processed"
 archetypeDir: "archetypes"
 layoutDir: "layouts"
-publishDi: "public"
```
Comment (bwplotka) [inline website/hugo.yaml]: wut

File: docs/contributing/how-to-contribute-to-docs.md [hunk 4]
```diff
@@ -63,6 +63,16 @@ Keep `menu` the same as sub-directory the file is in. This will help to manage a
...
 Show new menu section in main page by changing `website/layouts/_default/baseof.html` file.
 
+## Logos
+
+If your company is using Thanos in production, we'd love to showcase your company's logo on our page. Add yourself in [`website/data/sponsors.yml`](/website/data/sponsors.yml) like so:
+
+```yml
+- name: My Awesome Company
+  url: https://wwww.company.com
+  logo: company.png
```
Comment (domgreen) [inline docs/contributing/how-to-contribute-to-docs.md]: where do I put my file? Can we also ask for a specific resolution / size to ensure we dont have alot of back and forth?

File: website/hugo.yaml (hunk 3 above)
Comment (domgreen) [reply website/hugo.yaml]: `publishDir`?

File: website/layouts/index.html [hunk 5]
```diff
@@ -52,7 +52,33 @@ <h4 class="font-weight-bold">Downsampling & Compaction</h4>
...
+    </div>
+    {{ if $.Site.Data.sponsors.sponsors }}
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Used by</h3></div>
+    </div>
+    <div class="row">
+        {{ range $sponsor := $.Site.Data.sponsors.sponsors }}
+        <div class="col-4 col-xl-2 py-3">
+            <div class="img-sponsor">
+                <a href="{{ $sponsor.url }}" rel="nofollow" target="_blank"><img src="{{ (printf "/logos/%s" $sponsor.logo) | relURL }}" alt="{{ $sponsor.name }}"></a>
```
Comment (domgreen) [inline website/layouts/index.html]: what will happen if the logo is not present? Maybe have a default Thanos image to put there?

File: website/hugo.yaml (hunk 3 above)
Comment (adrien-f) [reply website/hugo.yaml]: Diff is weird here ? I've fixed the typo

File: website/layouts/index.html (hunk 5 above)
Comment (adrien-f) [reply website/layouts/index.html]: Added a check for the logo property !

File: website/layouts/index.html (hunk 1 above)
Comment (adrien-f) [reply website/layouts/index.html]: How about 
image ?

File: website/hugo.yaml (hunk 3 above)
Comment (bwplotka) [reply website/hugo.yaml]: just was curious why it is working on master then, that's fine

File: Makefile [hunk 6]
```diff
@@ -34,7 +34,7 @@ GIT               ?= $(shell which git)
 BZR               ?= $(shell which bzr)
 
 WEB_DIR           ?= website
-WEBSITE_BASE_URL  ?= thanos.io
```
Comment (bwplotka) [inline Makefile]: Please rebase ):

File: docs/contributing/how-to-contribute-to-docs.md [hunk 7]
```diff
@@ -63,11 +63,26 @@ Keep `menu` the same as sub-directory the file is in. This will help to manage a
 
 Show new menu section in main page by changing `website/layouts/_default/baseof.html` file.
 
+## Logos
```
Comment (bwplotka) [inline docs/contributing/how-to-contribute-to-docs.md]: Isn't that too hidden? Just curious how we can make it more searchable. Maybe some hover over logos "Go there " etc?

File: docs/contributing/how-to-contribute-to-docs.md (hunk 4 above)
Comment (bwplotka) [reply docs/contributing/how-to-contribute-to-docs.md]: Done I think.

File: website/layouts/index.html (hunk 1 above)
Comment (bwplotka) [reply website/layouts/index.html]: Yes! We can drop this when we have lot's of those ? And let's do it just very grey almost transparent?

**Comment by bwplotka** [comment]
Awesome thanks for doing it. There is legit CI error though.

File: docs/contributing/how-to-contribute-to-docs.md (hunk 7 above)
Comment (bwplotka) [reply docs/contributing/how-to-contribute-to-docs.md]: Ok, let's ignore my comment. It's fine to this being there we can just redirect people if they will aks to be on page (:

**Comment by domgreen** [comment]
personally would have the logo bigger under the `Founded by` section :+1: It is drowned by the size of the text above

**Comment by bwplotka** [comment]
And other logos as well bigger in same manner I think?

**Comment by bwplotka** [comment]
Let's all logos placeholder bit bigger, add some initial ones and announce rdy place for logos first then (: I want to have people pushing PRs, but we need to merge this first

**Pull request merged**